mod/localization/cui_zh_hk.sql

```diff
@@ -6,160 +6,158 @@ INSERT OR REPLACE INTO LocalizedText (Language, Tag, Text) VALUES
 
 -- =============================================================================
 -- City Panel
-("zh_Hans_HK", "LOC_CUI_CP_MANAGE_CITIZENS_TILES",                              "市民与单元格管理"),
+("zh_Hant_HK", "LOC_CUI_CP_MANAGE_CITIZENS_TILES", "市民與單元格管理"),
```

Comment:
For zh_Hans_HK, the 's' stand for simplified Chinese. So i change to zh_Hant_HK, and the 't' stand for traditional Chinese.

Reply:
These language codes are the code that the game uses to distinguish languages and cannot be modified.
I think the game uses "zh_Hans_HK" for traditional Chinese, because I didn't make that up, I copied it from original game language file.
For that reason, I'll change them back to "zh_Hans_HK", if you have issues seeing traditional Chinese in game, please let me know.
Other than that, it's all good.
Again, thank you very much ~!

Reply:
When i use "zh_hans_hk", the language of the mod will become German instead of Traditional Chinese. Therefore i try to change it to "zh_hant_hk". 

```
    <Replace Tag="LOC_BUILDING_TLACHTLI_NAME" Language="zh_Hant_HK">
      <Text>蹴球場</Text>
    </Replace>
```
I found out that in the Civ 6 game folder (\steamapps\common\Sid Meier's Civilization VI\Base\Assets\Text\Vanilla_zh_Hant_HK.xml), the xml use "zh_Hant_HK" as the language code for traditional Chinese.

And "zh_Hans_CN"  is for Simplified Chinese

Reply:
thanks for the reply, they must have change that in some updates, I'll upload a new version,
again, thanks~!